app/models/customer.rb

```diff
@@ -1,3 +1,3 @@
 class Customer < ApplicationRecord
-
+  has_many :invoices, dependent: :destroy
```

Comment:
the dependent: :destroy line ensures that the invoice that corresponds to the customer is deleted when the customer is deleted, correct?

Reply:
Yes, we weren't sure if we wanted to link the one to many with destroy's long term. Something to ask at the check-in.